### PR TITLE
Adjustments

### PR DESCRIPTION
**Pull request by 2can**
This pull request implements modifications to the current codebase, enhancing the reusability of `mdx` files across diverse platforms. These changes are the outcome of a proof of concept (POC) aimed at transitioning away from Docusaurus.

### CODE REVIEW

**Review by zaiste — APPROVED** [review]

File: docs/api-usage/authentication.mdx [hunk 1]
```diff
@@ -642,7 +642,7 @@ Saleor Apps extending the dashboard will be passed a special JSON Web Token (JWT
 
 You can also use this token to authenticate browser calls to your application's API by validating the token in your backend. See the user authentication section for more details on obtaining the necessary JWKS.
 
-See the [Saleor SDK documentation](https://github.com/saleor/saleor-app-sdk/blob/main/docs/app-bridge.md#events) for more details on how to receive the token.
+See the [Saleor SDK documentation](https://github.com/saleor/saleor-app-sdk/blob/main/docs/app-bridge.mdx#events) for more details on how to receive the token.
```
Comment (krzysztofwolski) [inline docs/api-usage/authentication.mdx:645]: The link no longer exists. Should be moved to https://saleor-docs-git-adjustments-saleorcommerce.vercel.app/docs/3.x/developer/extending/apps/developing-apps/app-sdk/app-bridge#events

File: versioned_docs/version-3.x/developer/extending/apps/architecture/app-bridge.mdx [hunk 2]
```diff
@@ -24,10 +24,10 @@ Events are sent from Dashboard, and App can listen to them.
 
 With events, the App can react, e.g., to theme changes or receive a token with a handshake event.
 
-You can find all events [here](https://github.com/saleor/saleor-app-sdk/blob/main/docs/app-bridge.md#events).
+You can find all events [here](https://github.com/saleor/saleor-app-sdk/blob/main/docs/app-bridge.mdx#events).
```
Comment (krzysztofwolski) [inline versioned_docs/version-3.x/developer/extending/apps/architecture/app-bridge.mdx:27]: Same as in current docs - the link no longer exists

**Review by krzysztofwolski — CHANGES_REQUESTED** [review]
Please update the old link to the docs in the repo. 
Besides that, I'll accept the PR

File: versioned_docs/version-3.x/developer/extending/apps/architecture/app-bridge.mdx (hunk 2 above)
Comment (2can) [reply versioned_docs/version-3.x/developer/extending/apps/architecture/app-bridge.mdx:27]: @krzysztofwolski done

**Review by krzysztofwolski — APPROVED** [review]

File: docs/developer/app-store/legacy-plugins/oidc.mdx [hunk 3]
```diff
@@ -23,7 +23,7 @@ Go to Configuration -> Plugins -> OpenID Connect and fill in the fields:
 - **User info URL:** The URL which can be used to fetch user details by using an access token (Required for flow: _Saleor as a resource server_).
 - **OAuth logout URL:** The logout URL which Saleor will return for frontend's logout request (Optional for flow: _Saleor as a client for authorization server_).
 - **Audience:** The Oauth resource identifier. If provided, Saleor will define the audience for each authorization request. Used to fetch user permissions from OAuth provider and map it to Saleor's permission (Optional for both integration flows).
-- **Use OAuth scope permissions:** Use OAuth scope permissions to grant a logged-in user access to protected resources. Your OAuth provider needs to have defined Saleor's permission scopes in format \_saleor:<saleor-permission\> (Required for both integration flows).
+- **Use OAuth scope permissions:** Use OAuth scope permissions to grant a logged-in user access to protected resources. Your OAuth provider needs to have defined Saleor's permission scopes in format \_saleor:&lt;saleor-permission\&lt; (Required for both integration flows).
```
Comment (patrys) [inline docs/developer/app-store/legacy-plugins/oidc.mdx:26]: The backslashes look wrong. It looks like they were meant to escape the lt/gt entities, but they are in random positions now.

**Comment by patrys** [comment]
What does transitioning away from Docusaurus mean?

**Comment by 2can** [comment]
Next.js based docs

File: docs/developer/app-store/legacy-plugins/oidc.mdx (hunk 3 above)
Comment (2can) [reply docs/developer/app-store/legacy-plugins/oidc.mdx]: Fixed

**Pull request merged**